proto/engine/v1/json_marshal_unmarshal.go

```diff
@@ -200,12 +201,12 @@ func (e *ExecutionPayload) UnmarshalJSON(enc []byte) error {
 		return err
 	}
 	*e = ExecutionPayload{}
-	e.ParentHash = dec.ParentHash
-	e.FeeRecipient = dec.FeeRecipient
-	e.StateRoot = dec.StateRoot
-	e.ReceiptsRoot = dec.ReceiptsRoot
-	e.LogsBloom = dec.LogsBloom
-	e.Random = dec.Random
+	e.ParentHash = bytesutil.PadTo(dec.ParentHash, 32)
+	e.FeeRecipient = bytesutil.PadTo(dec.FeeRecipient, 20)
+	e.StateRoot = bytesutil.PadTo(dec.StateRoot, 32)
+	e.ReceiptsRoot = bytesutil.PadTo(dec.ReceiptsRoot, 32)
+	e.LogsBloom = bytesutil.PadTo(dec.LogsBloom, 256)
+	e.Random = bytesutil.PadTo(dec.Random, 32)
```

Comment:
Have you considered reading `ssz-size` from the struct tag data?

Reply:
Good reminder, we have params configs for this. I'm going to switch to using constants instead of magic numbers. Do you think `ssz-size` is still preferred over config constants?